MAUI/TreeMap/migration.md

```diff
@@ -0,0 +1,1450 @@
+
+</table>
+
+### TreeMapTextStyle
+The `FontColor`, `FontSize`, `FontFamily`, and `FontAttributes` properties of the `SfTreeMap` is grouped to `TreeMapTextStyle`.
+
+<table>
+<tr>
+<th>Xamarin SfTreeMap control</th>
+<th>.NET MAUI SfTreeMap control</th>
+<th>Description</th>
+</tr>
+
+<tr>
+<td>{{'[Color](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfTreeMap.XForms.Style.html#Syncfusion_SfTreeMap_XForms_Style_Color)'| markdownify }}</td>
+<td>TextColor</td>
+<td>Gets or sets the text color for the SfTreeMap control.</td>
+</tr>
+
+<tr>
+<td>{{'[FontSize](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfTreeMap.XForms.Style.html#Syncfusion_SfTreeMap_XForms_Style_FontSize)'| markdownify }}</td>
+<td>FontSize</td>
+<td>Gets or sets the double value that represents the font size of the SfTreeMap control.</td>
+</tr>
+
+<tr>
+<td>{{'[FontFamily](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfTreeMap.XForms.Style.html#Syncfusion_SfTreeMap_XForms_Style_FontFamily)'| markdownify }}</td>
+<td>FontFamily</td>
+<td>Gets or sets the string, that represents font family of the SfTreeMap control.</td>
+</tr>
+
+<tr>
+<td>{{'[FontAttributes](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfTreeMap.XForms.Style.html#Syncfusion_SfTreeMap_XForms_Style_FontAttributes)'| markdownify }}</td>
+<td>FontAttributes</td>
+<td>Gets or sets the FontAttributes of the SfTreeMap control.</td>
+</tr>
+
+</table>
+
+## Enum
+
+<table>
+<tr>
+<th>Xamarin SfTreeMap control</th>
+<th>.NET MAUI SfTreeMap control</th>
+<th>Description</th>
+</tr>
+
+<tr>
+<td>{{'[LabelOverflowMode](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfTreeMap.XForms.LabelOverflowMode.html)'| markdownify }}</td>
+<td>TextFormatOption</td>
+<td>Specifies text formatting options for tree map leaf items in the SfTreeMap.</td>
+</tr>
+
+<tr>
+<td>{{'[LayoutTypes](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfTreeMap.XForms.LayoutTypes.html)'| markdownify }}</td>
+<td>LayoutType</td>
+<td>Specifies the layout type of algorithm used to arrange the items in the SfTreeMap.</td>
+</tr>
+
+<tr>
+<td>{{'[LegendPositions](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfTreeMap.XForms.LegendPositions.html)'| markdownify }}</td>
+<td>LegendPlacement</td>
+<td>Represents the legend placement to customize the legend position.</td>
+</tr>
+
+<tr>
+<td>{{'[LegendIcons](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfTreeMap.XForms.LegendIcons.html)'| markdownify }}</td>
+<td>LegendIconType</td>
+<td>Specifies the shape of icons used in legends for the SfTreeMap.</td>
+</tr>
+
+<tr>
+<td>{{'[SelectionMode](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfTreeMap.XForms.SelectionMode.html)'| markdownify }}</td>
+<td>SelectionMode</td>
+<td>Specifies the selection mode for the tree map items in the SfTreeMap.</td>
+</tr>
+
+</table>
+
+## Event
+
+<table>
+<tr>
+<th>Xamarin SfTreeMap control</th>
+<th>.NET MAUI SfTreeMap control</th>
+<th>Description</th>
+</tr>
+
+<tr>
+<td>Nil</td>
```

Comment:
Remove `nil.` As ae have ItemSelected evetargs in xamarin forms tree map.

Refer - https://help.syncfusion.com/cr/xamarin/Syncfusion.SfTreeMap.XForms.SfTreeMap.TreeMapItemSelectedEventHandler.html

Reply:
Added related API